STUDIO/org.openl.rules.jackson/src/org/openl/rules/serialization/JsonUtils.java

```diff
@@ -12,34 +15,30 @@
     private JsonUtils() {
     }
 
-    public static ObjectMapper createJacksonObjectMapper(Class<?>[] types, boolean enableDefaultTyping) {
+    public static ObjectMapper createJacksonObjectMapper(Class<?>[] types, boolean enableDefaultTyping) throws ClassNotFoundException {

```

Comment:
Changing the method signature may lead to potential compilation error and migrations on client projects which are using this class in groovy scripts cause they are not expecting checked exceptions

Reply:
Ignore ClassNotFoundExceprion and get NullPointerExeption next look like not correct behavior. @mkamalov your thoughts?

Reply:
I don't really want to fix potential compilation errors for all integration for all clients and would like to keep old behaviour